src/deprecated.jl

```diff
@@ -18,8 +18,8 @@ function DataFrame(column_eltypes::AbstractVector{T}, cnames::AbstractVector{Sym
     end
     for i in eachindex(categorical)
         categorical[i] || continue
-        elty = CategoricalArrays.catvaluetype(nonmissingtype(updated_types[i]),
-                                              CategoricalArrays.DefaultRefType)
+        elty = CategoricalValue{nonmissingtype(updated_types[i]),
```

Comment:
this deprecation is 1 year old - maybe just remove this method from deprecated.jl?

Reply:
You know I love keeping old deprecated methods. :-) I'd say let's remove them all at once later if we want.